tcpproxy.go

```diff
@@ -0,0 +1,104 @@
+// Copyright 2015 Canonical Ltd.
+// Licensed under the LGPLv3, see LICENCE file for details.
+
+package testing
```

Comment:
Does it make use of anything in testing or can it stand in its own package under testing?